dataconnect/README.md

```diff
@@ -0,0 +1,73 @@
+Firebase DataConnect Quickstart
+=======================================
+
+Introduction
+------------
+
+This is a sample app for the preview version of the Firebase DataConnect.
+This service is currently in Private Preview at no cost for a limited time. Sign up for the program at [Firebase Data Connect](https://firebase.google.com/products/data-connect).
+This quickstart will not work if you don't have access to the preview.
+
+<!-- Introduction
+------------
+
+[Read more about Firebase DataConnect ](https://firebase.google.com/docs/dataconnect/) -->
+
+#  Getting Started with Firebase Data Connect
+---------------
+Follow these steps to get up and running with Firebase Data Connect. For more detailed instructions, check out the [official documentation](https://firebase.google.com/docs/data-connect/quickstart).
+
+### 1. Create a New Data Connect Service and Cloud SQL Instance
+
+1. Open [Firebase Data Connect](https://console.firebase.google.com/u/0/project/_/dataconnect) in your project in Firebase Console and select Get Started.
+2. Create a new Data Connect service and a Cloud SQL instance. Ensure the Blaze plan is active. Pricing details can be found at [Firebase Pricing](https://firebase.google.com/pricing).
+3. Select your server region, if you wish to use vector search, make sure to select `us-central1` region.
+4. Allow some time for the Cloud SQL instance to be provisioned. After it's provisioned, the instance can be managed in the [Cloud Console](https://pantheon.corp.google.com/sql).
+
+### 2. Set Up Firebase CLI
+
+Ensure the Firebase CLI is installed and up to date:
+
+```bash
+npm install -g firebase-tools
+```
+
+### 3. Cloning the repository
+This repository contains the quickstart to get started with the functionalities of Data Connect.
+
+1. Clone this repository to your local machine.
+2. Enable the DataConnect CLI with the command `firebase experiments:enable dataconnect`.
```

Comment:
this step is no longer needed, I will update mine as well.

Reply:
That's right. We remove the need to enable experiments